docs/chapters/workflow/release/release-process.rst

```diff
@@ -32,6 +32,8 @@ List of artifacts
     - Instructions or pointers to instructions that have changed as a result of
       the changes since last release.
     - This list of artifacts.
+- Binary Wic images of all the variants for all the supported platforms.
+- The source code of all the packages released image contains.
```

Comment:
I think this sentence is a little bit confusing because it is missing a "the".
"The source code of all the packages the released image contains"?

Reply:
iz it betta nav?